CHANGELOG.md

```diff
@@ -52,38 +52,60 @@
 
 
 
-- **Enable Thread Instrumentation by default**
+- **Enable Thread instrumentation by default**
 
-  The configuration option `instrumentation.thread.tracing` is now enabled by default. This will allow the agent to properly monitor code that occurs inside of threads. If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists. [PR#1767](https://github.com/newrelic/newrelic-ruby-agent/pull/1767)
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code occurring inside threads. In Ruby agent 9.0, instrumented code within threads will be recorded and associated with the current transaction when the thread was created.
 
-- **Add Fiber Instrumentation**
+  This may be a breaking change if you are currently using custom thread instrumentation. New transactions inside of threads will no longer be started if one already exists. [PR#1767](https://github.com/newrelic/newrelic-ruby-agent/pull/1767)
+
+- **Add Fiber instrumentation**
 
   `Fiber` instances are now automatically instrumented similarly to `Thread` instances. This can be configured using `instrumentation.fiber`. [PR#1802](https://github.com/newrelic/newrelic-ruby-agent/pull/1802)
```

Comment:
I don't know how it looks on the docs website, but I think GitHub now supports links within single back tick code snippets. 

All this is to say, what do you think about linking the config options to their location on our config doc?

Reply:
I'll have to look into the single back tick. But the [config option is now linked](https://github.com/newrelic/newrelic-ruby-agent/pull/1827/commits/69c5455a3b26b45b67884d14aa16cf487628e5a4). Note: the docs site isn't live yet so the link doesn't work quite yet.